core/offchain/src/api.rs

```diff
@@ -37,17 +38,24 @@ enum ExtMessage {
 /// Asynchronous offchain API.
 ///
 /// NOTE this is done to prevent recursive calls into the runtime (which are not supported currently).
-pub(crate) struct AsyncApi(mpsc::UnboundedSender<ExtMessage>);
+pub(crate) struct Api<S> {
+	sender: mpsc::UnboundedSender<ExtMessage>,
+	db: S,
+}
 
 fn unavailable_yet<R: Default>(name: &str) -> R {
-	error!("This {:?} API is not available for offchain workers yet. Follow
+	error!("The {:?} API is not available for offchain workers yet. Follow \
 		   https://github.com/paritytech/substrate/issues/1458 for details", name);
 	Default::default()
 }
 
-impl OffchainExt for AsyncApi {
+const KEYS_PREFIX: &[u8] = b"keys";
+const STORAGE_PREFIX: &[u8] = b"storage";
```

Comment:
The idea for prefixes is to store keys in the same DB, but with a different prefix.

Reply:
What is the `KEYS_PREFIX` used for?

Reply:
Introduced by mistake, will be used in the future to store crypto keys :) Removed.